aviary/utils/propeller_map_conversion.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+
+import argparse
+import getpass
+import numpy as np
+from datetime import datetime
+from enum import Enum
+
+from aviary.utils.conversion_utils import conv_rep, conv_parse, read_map
+from aviary.api import NamedValues
+from aviary.utils.csv_data_file import write_data_file
+from aviary.utils.functions import get_path
+from aviary.subsystems.propulsion.utils import default_units
+
+
+class PropMapType(Enum):
+    GASP = 'GASP'
+
+    def __str__(self):
+        return self.value
+
+
+class PropModelVariables(Enum):
+    '''
+    Define constants that map to supported variable names in an propeller model.
+    '''
+    MACH = 'Mach_Number'
+    CP = 'CP'  # power coefficient
+    CT = 'CT'  # thrust coefficient
+    J = 'J'  # advanced ratio
+
+    @classmethod
+    def keys(cls):
+        return [c for c in cls]
+
+
+default_units = {
+    PropModelVariables.MACH: 'unitless',
+    PropModelVariables.CP: 'unitless',
+    PropModelVariables.CT: 'unitless',
+    PropModelVariables.J: 'unitless',
+}
+
+MACH = PropModelVariables.MACH
+CP = PropModelVariables.CP
+CT = PropModelVariables.CT
+J = PropModelVariables.J
+
+# gasp_keys = [MACH, CP, CT, J]
+
+header_names = {}
+for k in PropModelVariables.keys():
+    header_names[k] = k.value
```

Comment:
we can remove this by referencing `key.value` below
```suggestion
```

Reply:
True. Done.